client/src/theme/breakpoints.ts

```diff
@@ -1,7 +1,7 @@
 export const breakpoints = {
```

Comment:
use [`createBreakpoints`](https://chakra-ui.com/docs/theming/theme#breakpoints)?

Reply:
Sorry, this is my bad, I didn't use `createBreakpoints` initially. Would be a good idea based on the docs haha

Reply:
I can help fix this?